fe/fe-core/src/main/java/com/starrocks/catalog/TableProperty.java

```diff
@@ -76,6 +77,9 @@ public class TableProperty implements Writable, GsonPostProcessable {
 
     private boolean hasForbitGlobalDict = false;
 
+    @SerializedName(value = "storageInfo")
+    private StorageInfo storageInfo;
```

Comment:
StorageInfo is stored both in TableProperty and PartitionInfo. Isn't it a bit redundant

Reply:
This is the default storage info, you can use it when adding partition without cache property.